src/Directory.Build.props

```diff
@@ -9,6 +9,7 @@
     <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
     <RestoreLockedMode Condition="'$(CI)' == 'true'">true</RestoreLockedMode>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <EnlistmentRoot>$(MSBuildThisFileDirectory.TrimEnd('\\'))</EnlistmentRoot>
```

Comment:
>EnlistmentRoot [](http://example.com/codeflow?start=5&length=14)

Is this needed?

Reply:
Yeah it's used in `Bicep.Types.Az.csproj`. Copied this approach from one of our internal repos.